src/index.js

```diff
@@ -22,14 +22,16 @@ export default function LookupPlugin ({ mapComponents = {}, mapProps = {} }) {
   }
 }
 
+export const loopElementsInSchema = (schema, fn) => schema.map(row => row.map(el => fn(el)))
+
 /**
  * Remap components in a schema
  * @param {Array} schema - The schema
  * @param {Object|Function} mapComponents
 * @returns {Array}
  */
 const mapComps = (schema, mapComponents) => {
-  return schema.map(el => {
+  return schema.map(row => row.map(el => {
```

Comment:
I think this could use the function that you created above `loopElementsInSchema`.

Reply:
Good catch, this one slipped